godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/controllers.yaml

```diff
@@ -1,6 +1,5 @@
 controller_list:
-  - name: ""
-    action_ns: joint_trajectory_action
+  - name: 'joint_trajectory_action'
+    action_ns: ''
```

Comment:
Thank you jorge for your suggestion.

I'm going to do these changes base on your suggestion.

1.For the irb2400_client.sh
<https://github.com/swri-robotics/godel/pull/1#discussion_r55381912>,  I'm
going to optimize like below:
   1). Set the master URI in the launch file.
   2). Set the rviz file using substitution arguments
<http://wiki.ros.org/roslaunch/XML>.

2. Add  brief description of "network mode"

About the warning notice, I didn't pay attention on the log recently, but I
did see some warning before, I will try some test later.



On Tue, Mar 8, 2016 at 9:37 AM, Jorge Nicho <notifications@github.com>
wrote:

> In
> godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/controllers.yaml
> <https://github.com/swri-robotics/godel/pull/1#discussion_r55376276>:
>
> > @@ -1,6 +1,5 @@
> >  controller_list:
> > -  - name: ""
> > -    action_ns: joint_trajectory_action
> > +  - name: 'joint_trajectory_action'
> > +    action_ns: ''
>
> I thought it was necessary to specify the *action_ns* . Do you get any
> warnings related to actions not being found?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swri-robotics/godel/pull/1/files#r55376276>.
>